.travis.yml

```
@@ -1,5 +1,10 @@
 language: java
 jdk: openjdk11
+before_cache:
+  - rm -f $HOME/.gradle/caches/modules-2/modules-2.lock
```

:x: according to https://docs.gradle.org/current/userguide/dependency_resolution.html#sub:cache_copy both the lock file and `gc.properties` files should be removed

> Do not copy the *.lock or gc.properties files if they exist.
```suggestion
  - rm -f $HOME/.gradle/caches/modules-2/modules-2.lock
  - rm -f $HOME/.gradle/caches/modules-2/gc.properties
```